### DISCUSSION TITLE
Is it Possible to Migrate from Zwave 700 Controller to Zwave 800 Controller?

### DISCUSSION THREAD

**Discussion by Botched1** [zwave-js/zwave-js#5259] [General] [answered]
This may be an ignorant question, but...

When I went from Zwave 500 (UZB-3) to Zwave 700 (UZB-7) I was able to do so without re-pairing via migrating in zwave js. Very cool, and saved a ton of time.

Is it possible to migrate from a Zwave 700 controller (UZB-7 in my case) to a Zwave 800 controller (Zooz ZST39 in my case)?

As they are different chips, etc, I wasn't sure if that is possible or not, so thought I would ask.

**Comment by kpine** [comment, marked as answer]
No, not yet. Follow issue #5257 for updates.
